### PR TITLE
Cohort builder UI polish

### PR DESCRIPTION
**Pull request by tjennison-google**
* Merge group filter controls into a single drop down.
* Color icon buttons to match other buttons.
* Add text to demographics button.
* Reorder toolbar buttons and add an icon to the review button.
* Move all group controls to the left and add dividers.
* Move criteria controls to the left and shrink to match text.
* Move add group/criteria buttons to the top. Add text to the "add criteria" button since it's a bit out of place.
* Add an empty placeholder to the demographics pane.

### CODE REVIEW

File: ui/src/cohort.ts [hunk 1]
```diff
@@ -44,15 +44,36 @@ export function groupName(group: tanagra.Group, index: number) {
...
 
-export function groupFilterKindLabel(kind: tanagra.GroupFilterKindEnum) {
-  switch (kind) {
-    case tanagra.GroupFilterKindEnum.Any:
-      return "Any";
-    case tanagra.GroupFilterKindEnum.All:
-      return "All";
-  }
+export enum UIGroupFilter {
+  AT_LEAST_ONE = "At least one",
```
Comment (marikomedlock) [inline ui/src/cohort.ts:48]: Is this supported on the backend?

File: ui/src/overview.tsx [hunk 2]
```diff
@@ -40,12 +40,12 @@ import { isValid } from "util/valid";
...
   generateCohortFilter,
   getCriteriaPlugin,
-  groupFilterKindLabel,
   groupName,
+  uiGroupFilterFromFilter,
 } from "./cohort";
 
 export function Overview() {
-  const [showDemographics, setShowDemographics] = useState(true);
+  const [showDemographics, setShowDemographics] = useState(false);
```
Comment (marikomedlock) [inline ui/src/overview.tsx:48]: What does this change do -- what state are you no longer using?

**Review by marikomedlock — APPROVED** [review]

File: ui/src/cohort.ts (hunk 1 above)
Comment (marikomedlock) [reply ui/src/cohort.ts:48]: Nvm, I get it. This maps to `OR`.

File: ui/src/overview.tsx (hunk 2 above)
Comment (tjennison-google) [reply ui/src/overview.tsx:48]: This was just changing the default. Per the review I'll revert this back to defaulting open though.

**Pull request merged**